src/main/java/org/jitsi/videobridge/IceUdpTransportManager.java

```diff
@@ -1205,7 +1205,7 @@ private void describe(
      */
     private void describeDtlsControl(IceUdpTransportPacketExtension transportPE)
     {
-        DtlsControlImpl dtlsControl = getDtlsControl(/* channel */ null);
+        DtlsControlImpl dtlsControl = getSrtpControl(/* channel */ null);
```

Comment:
i'm surprised this works without a cast, shouldn't this store the result in an `SrtpControl` (and then check for `instanceof DtlsControlImpl`)?